apps/frontend/src/components/Class/index.tsx

```diff
@@ -136,8 +137,8 @@ export default function Class({
   const [updateUser] = useUpdateUser();
 
   const { data: course, loading: courseLoading } = useReadCourse(
-    providedClass?.subject ?? (subject as string),
-    providedClass?.courseNumber ?? (courseNumber as string)
+    providedClass?.subject || (subject as string),
```

Comment:
These were `??` because `??` is only truthy for `null` and `undefined` values. `||` is truthy for all truthy values like empty strings and 0. Changing this unnecessary reduces the strictness of this check